### ISSUE TITLE
Silent Task Notifier.

### ISSUE THREAD

**Issue by Sabra-xD** [CCExtractor/taskwarrior-flutter#330]
### Describe your improvement idea or feature request

Once a user sets up a task, we should schedule a notifier, silent one most probably, that is triggered 15 minutes as default before the task is due to alert the user that there is an upcoming task before it is due. 
We should also include it in the application settings so the user can turn it on and off.

### How would you like to see it implemented?

N/A

### Put here any screenshots or videos (optional)

N/A

### How can we contact you (optional)

N/A

### Would you like to work on this issue?

Yes

### By submitting this issue, I have confirmed that:

- [X] I have starred the repo ⭐ and watched 👀 it on GitHub and followed the contribution guidelines.

**Comment by Gaurav-Kushwaha-1225** [comment]
Is this issue still open??
@Sabra-xD @Pavel401

**Comment by BrawlerXull** [comment]
Feel free to work on this issue and send a pr!

**Comment by SahilDWanjare** [comment]
I agree, I done it in a wrong way. Actually I am new to open source and just started my journey as a contributer but Cauz of lack of guidance I ended with this 😔. Sorry sir, I want guidance how to do all this... I was just following the path shown by AI.

**Comment by Manar-Elhabbal7** [comment]
I can work on this issue
